spatialos-sdk/src/worker/parameters.rs

```diff
@@ -218,45 +129,27 @@ impl NetworkParameters {
     pub fn default() -> Self {
         NetworkParameters {
             use_external_ip: false,
-            protocol_params: ProtocolType::Tcp(TcpNetworkParameters::default()),
+            protocol: ProtocolType::Tcp(TcpNetworkParameters::default()),
             connection_timeout_millis: u64::from(WORKER_DEFAULTS_CONNECTION_TIMEOUT_MILLIS),
             default_command_timeout_millis: WORKER_DEFAULTS_DEFAULT_COMMAND_TIMEOUT_MILLIS,
         }
     }
 
     pub(crate) fn to_worker_sdk(&self) -> Worker_NetworkParameters {
-        let (protocol_type, raknet_params, tcp_params, kcp_params, modular_udp_params) =
-            self.protocol_params.to_worker_sdk();
         Worker_NetworkParameters {
             use_external_ip: self.use_external_ip as u8,
-            connection_type: protocol_type,
-            raknet: raknet_params,
-            tcp: tcp_params,
-            kcp: kcp_params,
-            modular_udp: modular_udp_params,
+            connection_type: 0,
```

Comment:
Can we add an comment here saying that we will fill in these parameters after this call? To match up with the comments in `do_connect`